### PR TITLE
[6.16] add test for bulk Hostgroup change

### PR DESCRIPTION
**Pull request by sambible**
Testing bulk Hostgroup change feature in All Hosts UI.

Requires: https://github.com/SatelliteQE/airgun/pull/1523

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_all_hosts_bulk_hostgroup_assignment'
airgun: 1523

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 8191
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_host.py -k test_all_hosts_bulk_hostgroup_assignment --external-logging
Test Result : ========== 1 passed, 70 deselected, 81 warnings in 754.20s (0:12:34) ===========
```

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -2461,3 +2461,46 @@ def _cleanup():
...
+    ).create()
+    for _ in range(3):
+        target_sat.api.Host(
+            organization=module_org, hostgroup=hostgroup, location=module_location
+        ).create()
+    with target_sat.ui_session() as session:
+        session.organization.select(module_org.name)
+        session.location.select(module_location.name)
+        headers = session.all_hosts.get_displayed_table_headers()
+        if "Host group" not in headers:
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:2492]:
```suggestion
        if 'Host group' not in headers:
```

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -2461,3 +2461,46 @@ def _cleanup():
             assert (
                 host.run(f'dnf remove {" ".join(packages)} -y').status == 0
             ), 'Could not remove installed packages in a finalizer!'
+
+
+@pytest.mark.tier2
+def test_all_hosts_bulk_hostgroup_assignment(target_sat, module_org, module_location, new_host_ui):
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:2467]: Remove `new_host_ui` if not using anywhere
```suggestion
def test_all_hosts_bulk_hostgroup_assignment(target_sat, module_org, module_location):
```

**Review by vijaysawant — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_host.py (hunk 2 above)
Comment (vsedmik) [reply tests/foreman/ui/test_host.py:2467]: I think this fixture is used just to enable the new host UI on the SAT, so we need to keep it.

**Review by vsedmik — APPROVED** [review]
Pending Vijay's quote comment, otherwise looks good.

**Pull request merged**